src/components/Registers.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Category from './Category';
+
+const RegisterPanel = ({ registers }) => {
+  const rs = registers.map((reg, i) => (
+    <li key={`{register-${i.toString()}}`}>
```

Comment:
you don't need to explicitly convert numbers to string, you could just do `${i}`

Reply:
so eslint will not let you use an index directly because of that issue we were talking about earlier. 
 [see this for a good explanation](https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys)
I also think its ok to use indices in our case because we arent adding or removing elements from the array; the size is static. I also did toString() to get past the eslint error